src/components/sections/Cards/RitcherCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+type Level = {
+  id: string;
+  range: string;
+  label: string;
+  description: string;
+  color: string;
+};
+
+const levels: Level[] = [
+  {
+    id: 'light',
+    range: '2.0 a 3.0',
+    label: 'Ligero',
+    description:
+      'Puede estar entre 2.0 a 3.0 en la escala Ritcher, no es percibido ni causa daños en donde fue originado.',
+    color: 'bg-[#86FE8B]',
```

Comment:
4. For the colors let's use the utility class I created for the ritcher scale, you can find these colors on the `tailwind.config.ts` file